requirements.txt

```diff
@@ -0,0 +1,9 @@
+dbus-python==1.2.4
+gpg==1.13.1
```

Comment:
```suggestion
gpg==1.10.0
```

Reply:
per discussion in https://app.slack.com/client/T027F3GAJ/GJWNXSY4E/thread/GJWNXSY4E-1659344708.635199 seems like we need to set an older dummy version here.

Reply:
only for gpg, other versions are ok? 
@SkorikSergey could you please check that

Reply:
@svor It failed with next error:
<code>
 CMD :python -m pip install -r "/projects/python-hello-world/requirements.txt" && python - "/projects/python-hello-world/requirements.txt" "/projects/python-hello-world/target/pylist.json"
 STDOUT : Defaulting to user installation because normal site-packages is not writeable
Collecting dbus-python==1.2.4
  Downloading dbus-python-1.2.4.tar.gz (612 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 612.1/612.1 kB 18.4 MB/s eta 0:00:00
  Preparing metadata (setup.py): started
  Preparing metadata (setup.py): finished with status 'done'
Collecting gpg==1.10.0
  Downloading gpg-1.10.0.tar.gz (39 kB)
  Preparing metadata (setup.py): started
  Preparing metadata (setup.py): finished with status 'error'
 
 STDERR :   error: subprocess-exited-with-error
  
  × python setup.py egg_info did not run successfully.
  │ exit code: 1
  ╰─> [1 lines of output]
      Could not find gpgme-config.  Please install the libgpgme development package.
      [end of output]
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
error: metadata-generation-failed

× Encountered error while generating package metadata.
╰─> See above for output.

note: This is an issue with the package mentioned above, not pip.
hint: See above for details.
</code>